candidate-standard/clause_2_conformance.adoc

```diff
@@ -3,32 +3,52 @@ Conformance with this standard shall be checked using the tests specified in Ann
 
 The one Standardization Target for this standard is Web APIs.
 
-OGC API-Common provides a common foundation for OGC API standards. Therefore, this standard should be viewed as an extension to API-Common. Conformance to this standard requires demonstrated conformance to the applicable Conformance Classes of API-Common.
+OGC API-Common-Part 1: Core provides a common foundation for OGC API standards. Therefore, this EDR API standard should be viewed as an extension to API-Common-Core. Conformance to this standard requires demonstrated conformance to the applicable Conformance Classes of API-Common-Core.
 
-This standard identifies six (6) Conformance Classes. The Conformance Classes implemented by an API are advertised through the /conformance path on the landing page. Each Conformance Class is defined by one Requirements Class. The tests in Annex A are organized by Requirements Class. The Requirements Classes define the functional requirements which will be tested through the associated Conformance Class.
+This standard identifies seven (7) Conformance Classes. The Conformance Classes implemented by an API are advertised through the /conformance path on the landing page. Each Conformance Class is defined by one or more Requirements Classes. The tests in Annex A are organized by Requirements Class. The Requirements Classes define the functional requirements which will be tested through the associated Conformance Class.
 
-The requirements classes for OGC API-EDR are:
+The conformance classes and their constituent requirements classes for OGC API-EDR are:
 
-* <<rc_core-section,Core>>
+* <<rc_core-section,Core>>, one Requirement Class
 
 The _Core Requirements Class_ is the minimal useful service interface for an OGC API. The requirements specified in this Requirements Class are mandatory for all implementations of API-EDR.
 
-* <<rc_collections-section,Collections>>
+The Core requirements class is specified in *Requirements Class Core* <<rc_core-section,(Chapter 8)>>.
 
-NOTE: Validate that this description is accurate.
+* <<rc_collection-section,Collection>>, one Requirement Class
 
-The _Collections Requirements Class_ defines the requirements specific to the query and retrieval of Environmental Data Resources.
+The API-Common _Collections_ standard extends the API-Common _Core_ to enable discovery and query access to <<collection-definition,collections>> of <<spatial-resource-definition,spatial resources>>.
```

Comment:
Think this should be "spatial data collections". Or even just "spatial data". instal of "collections of spatial resources".